### PR TITLE
Merge all setup checks into one controller

### PR DESCRIPTION
**Pull request by MorrisJobke**
* renamed hasMissingIndexes to missingIndexes
* properly integrates the two sections into one
* having also severities depending on the results

## Before

bildschirmfoto 2018-06-13 um 15 03 08

Also it were two distinct sections:

bildschirmfoto 2018-06-13 um 15 06 00

## After

bildschirmfoto 2018-06-13 um 15 00 28

### CODE REVIEW

File: settings/templates/settings/admin/overview.php [hunk 1]
```diff
@@ -23,133 +23,13 @@
...
-		}
-
-		// Are doc blocks accessible?
-		if (!$_['isAnnotationsWorking']) {
-			?>
-			<li>
-				<?php p($l->t('PHP is apparently set up to strip inline doc blocks. This will make several core apps inaccessible.')); ?><br>
-				<?php p($l->t('This is probably caused by a cache/accelerator such as Zend OPcache or eAccelerator.')); ?>
-			</li>
-			<?php
```
Comment (MorrisJobke) [inline settings/templates/settings/admin/overview.php]: Dropped this one, as it is already checked in base.php and if this does not work you will never see the admin page anyways.

File: settings/templates/settings/admin/overview.php [hunk 2]
```diff
@@ -23,133 +23,13 @@
...
-				p($l->t('System locale can not be set to a one which supports UTF-8.'));
-				?>
-				<br>
-				<?php
-				p($l->t('This means that there might be problems with certain characters in filenames.'));
-				?>
-				<br>
-				<?php
-				p($l->t('It is strongly proposed to install the required packages on your system to support one of the following locales: %s.', [$locales]));
-				?>
```
Comment (MorrisJobke) [inline settings/templates/settings/admin/overview.php]: Also dropped this for the same reason.

**Review by skjnldsv — APPROVED** [review]
Make sense!!

**Review by rullzer — APPROVED** [review]
I like it!

**Pull request merged**